app/controllers/wpcc_controller.rb

```diff
@@ -0,0 +1,17 @@
+class WpccController < EmbeddedToolsController
+  before_action :authenticate
+
+  protected
+
+  def authenticate
+    authenticate_or_request_with_http_basic do |username, password|
+      Authenticable.authenticate(username, password)
+    end
+  end
+
+  def category_id
+    ''
+  end
```

Comment:
This is a remnant of #1768 branch. Good spot. The category_id, It is for the breadcrumbs. I will ask Umair if is an acceptance criteria of this user story and in case it doesn't I will leave as it is.